### ISSUE TITLE
2.0 NER training - old labels don't come up after adding new labels

### ISSUE THREAD

**Issue by BeardyBear** [explosion/spaCy#1496]
Hello.

I am playing with new version of "Training an additional entity type" snippet from docs.
After updating a `en_core_web_sm` or any other pre-trained model with a new label (e.g. ANIMAL as per example) for any number of epochs, the model stops to identify the labels it was able to identify before (LOC, PERSON, etc.), while ANIMAL or any other new label works just fine.

This behavior is consistent throughout all the alpha versions (current: a18)
Is it just work in progress and not supposed to work yet?
Or maybe I misunderstand something.

Thank you.
Spacy is awesome and I am very hyped for 2.0 release.

**Comment by ines** [comment]
Hi! I think you might be hitting the [the "catastrophic forgetting" problem](https://explosion.ai/blog/pseudo-rehearsal-catastrophic-forgetting) (also see #1318, #1382 and #1337). If your training data only contained new entities and you didn't mix in any examples the model previously recognised, this might have caused the model to "forget" what it had previously learned. The solution to this would be to pre-label some text, and mix it with the new text in your updates. You can also do this by running spaCy over some text, extracting a bunch of non-ANIMAL entities you care about, and adding them to your training examples.

We should make this more explicit in the docs and add a note to the code examples.

**Comment by BeardyBear** [comment]
Thanks @ines, that might very well be the cause - I will definitely try to mix up the training data.

**Comment by ines** [comment]
Cool – I hope this solves the problem. If not, feel free to reopen!
